src/lib/calculation.ts

```diff
@@ -136,74 +149,80 @@ export function calculate(input: TaxFormUserInput): TaxForm {
       return this.r055
     },
     get r072_pred_znizenim() {
-      return this.r057 + this.r040
+      return this.r057.add(this.r040)
     },
     get r073() {
-      const result =
-        this.r072_pred_znizenim > 20507 // TODO test both cases here
-          ? Math.max(0, 9064.094 - (1 / 4) * this.r072_pred_znizenim)
-          : Math.max(0, NEZDANITELNA_CAST_ZAKLADU)
-
-      return round2decimal(result)
+      return this.r072_pred_znizenim.gt(20507) // TODO test both cases here
+        ? Decimal.max(
+            0,
+            new Decimal(9064.094).sub(this.r072_pred_znizenim.times(1 / 4)),
+          )
+        : Decimal.max(0, NEZDANITELNA_CAST_ZAKLADU)
     },
     get r074_znizenie_partner() {
       if (this.r032_uplatnujem_na_partnera) {
-        const result =
-          this.r072_pred_znizenim > 36256.38
-            ? Math.max(
-                0,
-                (13001.438 -
-                  (1 / 4) * this.r072_pred_znizenim -
-                  Math.max(this.r032_partner_vlastne_prijmy, 0)) *
-                  (1 / 12) *
-                  this.r032_partner_pocet_mesiacov,
-              )
-            : Math.max(
-                0,
-                (3937.35 - Math.max(this.r032_partner_vlastne_prijmy, 0)) *
-                  (1 / 12) *
-                  this.r032_partner_pocet_mesiacov,
-              )
-        return round2decimal(result)
+        return this.r072_pred_znizenim.gt(36256.38)
+          ? Decimal.max(
+              0,
+              new Decimal(13001.438)
+                .minus(
+                  this.r072_pred_znizenim
+                    .times(1 / 4)
```

Comment:
Namiesto nasobenia 1/4, co tak radsej videlit 4? Alebo nasobit 0.25? Ani 1/4 nemusi byt vzdy 0.25 :)

To iste aj nizsie s 1/12 (vidim to tam 2x).

Reply:
1/4 je v oficialnom vzorci, preto som to dal takto ale mas pravdu